src/trans/gpu/internal/trgtol_mod.F90

```diff
@@ -1482,13 +1548,12 @@ SUBROUTINE TRGTOL(PGLAT,KF_FS,KF_GP,KF_SCALARS_G,KVSET,KPTRGP,&
 ! when run as gpnorm on cpu needs to avoid this update call
 IF(.NOT.LGPNORM) THEN
 #ifdef ACCGPU
-  !$ACC UPDATE DEVICE(PGLAT)
-#endif
-#ifdef OMPGPU
-  !$OMP TARGET UPDATE TO(PGLAT)
+!! All data is updated on device already. PJM 9/05/2023
+!  !$ACC UPDATE DEVICE(PGLAT)
```

Comment:
Hi @PaulMullowney - I see what you mean. I don't know the issue with LGPNORM. However one level up in ftdir_ctl_mod.F90, we _always_ update `ZGTF`:
```
#ifdef ACCGPU
!$ACC UPDATE DEVICE(ZGTF)
#endif
#ifdef OMPGPU
!$OMP TARGET UPDATE TO(ZGTF)
#endif
```

Is that what you're referring to?

Reply:
Any update on getting this PR reviewed, tested and merged?